gulpfile.js

```diff
@@ -100,14 +100,27 @@ gulp.task('swag-img:download', () => {
 });
 
 gulp.task('swag-img:optimize', () => {
+	const jpegOutputConfig = {
+		name: '*.jpeg',
+		height: 300,
+		flatten: true
+	};
+	const webpOutputConfig = Object.assign({
+		rename: i => `${i.basename}.webp`
+	}, jpegOutputConfig);
+
 	return gulp.src('dist/assets/swag-img/*')
-		.pipe(responsive([{
-			name: '*',
-			height: 300,
-			format: 'jpeg',
-			flatten: true
-		}], RESIZE_OPTS))
-		.pipe(gulp.dest('dist/assets/swag-img'));
+		.pipe(responsive([jpegOutputConfig, webpOutputConfig], RESIZE_OPTS))
+		.pipe(gulp.dest('dist/assets/swag-img'))
+		.on('end', () => {
```

Comment:
I'm pretty sure you need to use a callback if you add an `on('end')` listener because you run into race conditions